src/main/res/values/strings.xml

```diff
@@ -463,7 +463,7 @@
 
     <!-- 2013-05-18 -->
 
-    <string name="action_fling_disabled">Disabled</string>
+    <string name="action_fling_disabled">Off</string>
```

Comment:
Is this for inclusivity reasons? Maybe clearer as "No action".

Reply:
No, disabled people are still disabled. This presumably is either "On" or "Off".(?)

Reply:
This is an option under Behaviour -> Post Actions -> Fling left/right. "Disabled" is one of a series of options such as "Upvote", "Downvote", etc.

"Off" seems slightly weird in that list, "No action" could work but lacks the implication that flinging in that direction is prevented. So leaving this as "Disabled" is probably best.

Reply:
Disabled seems like it is beyond the control of the user to turn it on or off. Less complexity is good.